dbms/src/Interpreters/Join.h

```diff
@@ -407,11 +410,18 @@ class Join
     Sizes key_sizes;
 
     /// Block with columns from the right-side table except key columns.
-    Block sample_block_with_columns_to_add;
+    Block sample_block_without_keys;
     /// Block with key columns in the same order they appear in the right-side table.
-    Block sample_block_with_keys;
+    Block sample_block_only_keys;
 
-    Names tidb_output_column_names;
+    NamesAndTypes output_columns;
+    Block output_block;
+    NamesAndTypes output_columns_after_finalize;
+    Block output_block_after_finalize;
+    NameSet output_column_names_set_after_finalize;
+    NameSet output_columns_names_set_for_other_condition_after_finalize;
+    Names required_columns;
+    bool finalized = false;
```

Comment:
Can clickhouse-client execute join sql normally?

Reply:
No, join is not supported for sql from clickhouse-client. We already disable these sql since v7.0.0(https://github.com/pingcap/tiflash/pull/7064)

Reply:
ok